### PR TITLE
add elasticsearch 'exists'

### PR DESCRIPTION
**Pull request by AmerbankDavd**

### CODE REVIEW

**Comment by StarpTech** [comment]
Hi @AmerbankDavd, thanks for the PR. Could you add a test and update the docs ? :)

**Comment by AmerbankDavd** [comment]
It's my pleasure.
Test added, hope it's fine.

File: README.md [hunk 1]
```diff
@@ -77,6 +77,31 @@ hemera.act({
...
+* `data`:
+  * `index`: the name of your index `string`
+  * `type`: the type of your document
+  * `id`: the id of your document
+
+Example:
+```js
+hemera.act({
+  topic: 'elasticsearch',
+  cmd: 'search',
```
Comment (StarpTech) [inline README.md]: Please use exists signature

**Comment by StarpTech** [comment]
Please fix the example for `exists` then it's fine 👍

**Comment by StarpTech** [comment]
hemera-elasticsearch 3.0.2 published 🎉

**Comment by AmerbankDavd** [comment]
Thank you!

**Pull request merged**